deeposlandia/webapp.py

```diff
@@ -170,7 +171,7 @@ def prediction():
     utils.logger.info("file: {}, dataset: {}, model: {}".format(dataset, model,
                                                                 filename))
     predictions = predict([filename], "mapillary", "semantic_segmentation",
-                          aggregate=False, output_dir=PREDICT_FOLDER)
+                          aggregate=True, output_dir=PREDICT_FOLDER)
```

Comment:
is it possible to use the  `agg_value` variable previously defined?

Reply:
The `agg_value` is defined in another method, starting from the dataset passed as a parameter in the app.

Here we know that we want to predict Mapillary labels (this method is relative to "try-it-yourself" part of the app, where clients may upload their own images), so it makes sense to consider a hard-coded aggregate value (True, as it was chosen to consider this case in the PR).

Reply:
OK, thanks.